src/engraving/tests/CMakeLists.txt

```diff
@@ -66,10 +66,10 @@ set(MODULE_TEST_SRC
     ${CMAKE_CURRENT_LIST_DIR}/tst_splitstaff.cpp
     # ${CMAKE_CURRENT_LIST_DIR}/tst_text.cpp not actual, not compile
     ${CMAKE_CURRENT_LIST_DIR}/tst_timesig.cpp
-    # ${CMAKE_CURRENT_LIST_DIR}/tst_tools.cpp # fail
-    # ${CMAKE_CURRENT_LIST_DIR}/tst_transpose.cpp # fail
-    # ${CMAKE_CURRENT_LIST_DIR}/tst_tuplet.cpp # fail
-    # ${CMAKE_CURRENT_LIST_DIR}/tst_unrollrepeats.cpp # fail
```

Comment:
not sure why these were marked "fail", they all work fine (with one slight tweak needed for one of the TestTuplet tests)